src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerDriver.cs

```diff
@@ -89,6 +89,8 @@ internal abstract partial class AnalyzerDriver : IDisposable
         /// </summary>
         private ImmutableDictionary<DiagnosticAnalyzer, GeneratedCodeAnalysisFlags> _generatedCodeAnalysisFlagsMap;
 
+        private AnalyzerActions _analyzerActions;
```

Comment:
Please add a comment, everything else in the driver has one.

Reply:
@sharwell this was not addressed.

---
In reply to: [362572236](https://github.com/dotnet/roslyn/pull/40691#discussion_r362572236) [](ancestors = 362572236)

Reply:
Correct, comments in my pull requests that are not marked resolved have not been addressed unless you see a ➡️ reply with a stated follow-up action item.

Reply:
Comment is now added